### PR TITLE
Cleaned up SQL handling

### PR DESCRIPTION
**Pull request by NiLSPACE**
Instead of a huge wrapper I now have all queries pre-defined. This is quite allot smaller than it was previously, and much cleaner to read.

### CODE REVIEW

File: storage_sqlite.lua [hunk 1]
```diff
@@ -28,56 +35,50 @@ end
...
 
-function cSQLiteStorage:RegisterOrChangePassword(a_UUID, a_Password)
-	local whereList = cWhereList():Where('uuid', a_UUID)
-	local res = self.m_DB:Select("players", "*", whereList)
-	
-	if (not res) then
-		return false, "Something went wrong in the database"
+--- Executes a query that was loaded before.
+-- The parameters is a dictionary. The key is the name of the parameter. 
+-- This can be found with a $ or : before it in the actuall query.
```
Comment (madmaxoft) [inline storage_sqlite.lua]: `before` -> `in front of`
`actuall` -> `actual`

**Comment by madmaxoft** [comment]
This looks much better. Still, I'd like to ask about the reasoning for pulling the queries out into separate files. What advantage is there compared to alternatives?

**Comment by NiLSPACE** [comment]
It's just preference. I prefer to keep the queries separate from the actual code. I find it to be much more readable.

**Comment by madmaxoft** [comment]
It's a bit more work when editing them - you need to jump between various files in the editor.
I'd have initialized the `Queries` dictionary directly as a global in the Lua code in `storage_sqlite.lua`

**Comment by NiLSPACE** [comment]
You could just use the split screen function in Notepad++ or ZeroBrane.

I did that at first, but removed it for some reason. I've reverted that back now though :)

**Pull request merged**